### PR TITLE
Update the JRubyConf EU text

### PR DESCRIPTION
**Pull request by lucaspinto**
@myabc @PragTob WDYT

### CODE REVIEW

File: app/views/pages/guide.html.erb [hunk 1]
```diff
@@ -13,9 +13,9 @@ If necessary, differences are clearly marked as such.</p>
 <p>Rails talks are welcome, but we want to cover a broad range of topics, so those will be limited.</p>
 <p>We also accept non-talk proposals, like workshops and moderated discussions. You can select your format in the CFP process.</p>
 
-<h3 id="topics-jrubyconf">Topics for JRubyConf.EU</h3>
+<h3 id="topics-jrubyconf">Topics for JRubyConf EU</h3>
 
-<p>JRubyConf.EU is shorter this year than last year, so we ask you talk to be strictly involving JRuby. We are interested in development techniques, user stories and general outlook for the JRuby ecosystem.</a>
+<p>The JRubyConf EU is a one-day one-track conference and the only one worldwide 100% dedicated to JRuby. This obviously means that we’re interested in talks involving JRuby. We’re looking for topics regarding development techniques, user stories and general outlook for the JRuby ecosystem. Note that we sometimes pick a eurucamp proposal when it seems to us that it could be of interest to the JRubyConf EU audience.</p>
```
Comment (PragTob) [inline app/views/pages/guide.html.erb]: Looks good, but what does `user stories` mean? For me it ties too much to the concept of user stories in agile development. Perhaps "usage stories"? Or "success stories" ? Or in another style "How do you use JRuby in your company?" "What problem did JRuby help you solve?" "How do you setup JRuby to be most effective?" "What JRuby pro tips did you always want to share?" "What JRuby limitations do you know and how did you work around them?" "What cool library did you write and want everyone to know about?"

**Comment by PragTob** [comment]
Looks good, left comments inline :)

**Comment by lucaspinto** [comment]
I guess it's fine now.

File: app/views/pages/guide.html.erb [hunk 2]
```diff
@@ -13,9 +13,11 @@ If necessary, differences are clearly marked as such.</p>
 <p>Rails talks are welcome, but we want to cover a broad range of topics, so those will be limited.</p>
 <p>We also accept non-talk proposals, like workshops and moderated discussions. You can select your format in the CFP process.</p>
 
-<h3 id="topics-jrubyconf">Topics for JRubyConf.EU</h3>
-
-<p>JRubyConf.EU is shorter this year than last year, so we ask you talk to be strictly involving JRuby. We are interested in development techniques, user stories and general outlook for the JRuby ecosystem.</a>
+<h3 id="topics-jrubyconf">Topics for JRubyConf EU</h3>
+<p>The JRubyConf EU is a one-day one-track conference and the only one worldwide 100% dedicated to JRuby. This obviously means that we’re interested in talks involving JRuby.</p>
+<p> How do you use JRuby in your company? What problem did JRuby help you solve? How do you setup JRuby to be most effective? What JRuby pro tips did you always want to share? What JRuby limitations do you know and how did you work around them? What cool library did you write and want everyone to know about?</p>
```
Comment (bascht) [inline app/views/pages/guide.html.erb]: I'd drop the last 5 occurrences of »JRuby«, no?

**Pull request merged**